ZenPacks/zenoss/OpenStackInfrastructure/resources/global.js

```diff
@@ -127,17 +127,59 @@ var addOpenStack = new Zenoss.Action({
                         xtype: 'label',
                         style: 'font-style: italic',
                         text: '(NOVA_PROJECT_ID or OS_TENANT_NAME)',
-                        margin: '0 0 0 10'
+                        margin: '15 0 0 10'
                     }]
                 }, {
+                    xtype: 'container',
+                    layout: 'hbox',
+                    items: [{
+                        xtype: 'textfield',
+                        name: 'user_domain_name',
+                        fieldLabel: _t('User Domain Name'),
+                        labelWidth: 130,
+                        id: "openstack_user_domain_name",
```

Comment:
I haven't had time to actually try this out today, but does this show the default value ('default') in the UI?  It probably should.